apps/src/codebridge/Workspace/workspace.module.scss

```diff
@@ -5,7 +5,8 @@
   grid-area: workspace;
   background-color: $darkest_gray;
   color: $white;
-  overflow: scroll;
+  overflow: hidden;
+  display: block;
```

Comment:
The `PanelContainer` sets `display : flex` and we need to override it here to ensure that the FileTabs stay below the panel title when the `FileBrowser` and `Editor` are wrapped in a div. 🤷 

It's almost assuredly a hack and points to _something_ somewhere else needing to be tweaked in the CSS.

We also explicitly set overflow to hidden here, and explicitly set it to scroll on the `workspaceWorkarea` to ensure that we _only_ scroll in the editor. There may still be other scroll related headaches that pop up, so be on the lookout and file bugs if anything is discovered.

Reply:
Fixed!